src/apps/Plotter.jsx

```diff
@@ -0,0 +1,114 @@
+import functionPlot from "function-plot";
+import { useState } from "react";
+import styled from "styled-components";
+
+//------------Styled Components-------------
+
+const Form = styled.form`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+`;
+
+const Input = styled.input`
+  margin: 2px;
+`;
+
+const H3 = styled.h3`
+  color: red;
+`;
+
+//=========================================
+
+let contentsBounds = document.body.getBoundingClientRect();
+let width = 1500;
+let height = 500;
+let ratio = contentsBounds.width / width;
+width *= ratio;
+height *= ratio;
+
+function Renderer() {
+  const [text, changeText] = useState("x^2");
+  const [x_axis, changex] = useState(10);
+  const [y_axis, changey] = useState(5);
+  const [error, changeError] = useState("");
+
+  function handleSubmit(e) {
+    try {
+      e.preventDefault();
+      functionPlot({
+        target: "#instant_apps_root",
+        width,
+        height,
+        xAxis: { domain: [-x_axis, x_axis] },
+        yAxis: { domain: [-y_axis, y_axis] },
+        grid: true,
+        data: [
+          {
+            fn: text,
+          },
+        ],
+      });
+      changeError("");
+    } catch {
+      changeError("Wrong Input, Please Correct it...");
+    }
```

Comment:
Can we catch and show the exact error here? This may be possible this way: 

```suggestion
    } catch(error) {
      changeError(error);
    }
```

Reply:
Was thinking of this as a generic msg would be better for a user than a error msg.

Reply:
That's good thinking, but generic messages don't tend to helpful when it comes to understanding what our problem may be. We can have a "read more" expand icon, through which one can read the error message in more detail. What do you think? 

Also, what kind of error messages does the library show?

Reply:
![image](https://user-images.githubusercontent.com/68731551/135704383-0089efa8-2bed-403d-a878-b6689f65530e.png)
Logged the error msg

The error msg was basically the call stack that caused the error. and didn't find it much useful.

Reply:
Ah, I see. Let's go with the default error message then, thanks for trying this out. Also, we can remove the console.log or warn statements, we should keep them only for development and not commit them in production.

Reply:
Ok sure, removing the log statements...